Automattic/simplenote-electron
# Update resources

### Fix
This PR updates the certificate used to sign the Mac build that expired a few weeks ago.

### Test
1. Open the _artifact_ job on the CI for this branch.
2. Download the Mac OS build and install it.
3. Launch and app and verify that you are presented with the option to run it.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
-
